projects/packages/sync/src/modules/class-posts.php

```diff
@@ -434,6 +444,12 @@ public function filter_post_content_and_add_links( $post_object ) {
 			$post->post_password = 'auto-' . wp_generate_password( 10, false );
 		}
 
+		// Explicitly truncate post_content when it exceeds limit.
```

Comment:
When I see "truncate" I think that it will cut it off at the limit, not exclude it entirely. "Remove" or "omit" seem more descriptive.
```suggestion
		// Explicitly omit post_content when it exceeds limit.
```